packages/jellyfish-api-core/__tests__/category/blockchain/getMempoolAncestors.test.ts

```diff
@@ -0,0 +1,93 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+import BigNumber from 'bignumber.js'
+
+describe('getMempoolAncestors', () => {
+  const container = new MasterNodeRegTestContainer()
+  const testing = Testing.create(container)
+  const address = 'mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU'
+  const amount = 0.003
+
+  beforeAll(async () => {
+    await testing.container.start()
+    await testing.container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  it('should return empty array for transaction id without ancestors', async () => {
+    const txId = await testing.rpc.wallet.sendToAddress(address, amount)
```

Comment:
```suggestion
    const txId = await testing.rpc.wallet.sendToAddress('mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU', 0.003)
```

We should also take a different approach to writing theses. E.g. things like making them abstract or DRY might inherently make tests harder to read and digest.

It might be better to make them static so the reader of the test can follow the test without cross-referencing fields from top and bottom of the file